packages/visualizations/src/components/Map/Choropleth.svelte

```diff
@@ -1,18 +1,31 @@
 <svelte:options immutable={true} />
 
-<script>
+<script lang="ts">
     import turfBbox from '@turf/bbox';
+    import type { SourceSpecification } from 'maplibre-gl';
+    // eslint-disable-next-line import/no-unresolved
```

Comment:
Regarding this issue (type only imports), I seems that the version `2.25.2` of `eslint-plugin-import` fixed it. I've tested this version and no linter more error 👍.

Here's the PR: https://github.com/import-js/eslint-plugin-import/pull/2220

(same thing in [MapRender.svelte](https://github.com/opendatasoft/ods-dataviz-sdk/pull/100/files#diff-39ec5d7e71c54fa06eac98981d148eace0b3a0ea67e9a19ca65ca535b93a93a4) line 16)